openfl-docker/Dockerfile.workspace

```diff
@@ -8,13 +8,11 @@ FROM ${BASE_IMAGE}
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-USER openfl
+USER user
 ARG WORKSPACE_NAME
 COPY ${WORKSPACE_NAME}.zip .
-
-WORKDIR /home/openfl
 RUN fx workspace import --archive ${WORKSPACE_NAME}.zip && \
-    pip install -r ${WORKSPACE_NAME}/requirements.txt
+    pip install --no-cache-dir -r ${WORKSPACE_NAME}/requirements.txt
```

Comment:
Is the pip install necessary given that it is [already done by import sub-command](https://github.com/securefederatedai/openfl/blob/develop/openfl/interface/workspace.py#L253-L263)?

Reply:
Ideally, import should not install packages (#73). A separate install is explicit and allows users to supply additional flags (or even use different package managers).

Implicit pip install will be removed in a future PR.